src/generators/genjvm.ml

```diff
@@ -2249,7 +2249,8 @@ class tclass_to_jvm gctx c = object(self)
 			) c_int.cl_implements
 		in
 		loop (fun t -> t) c;
-		begin match c.cl_overrides,c.cl_super with
+		let overrides = List.filter (fun cf -> has_class_field_flag cf CfOverride) c.cl_ordered_fields in
```

Comment:
`cl_ordered_statics`?

Reply:
There are quite a few other places, where `cl_ordered_statics` is probably required too.

Reply:
Also, what about `cl_constructor`? I think they also could be overridden.

Reply:
Huh? Isn't this about overrides, not _overloads_?

Reply:
Yeah, I'm just always confused by these words similarity )